python-package/lightgbm/sklearn.py

```diff
@@ -847,6 +847,20 @@ def objective_(self):
             raise LGBMNotFittedError('No objective found. Need to call fit beforehand.')
         return self._objective
 
+    @property
+    def n_estimators_(self):
```

Comment:
```suggestion
    def n_estimators_(self) -> int:
```

I think using return type hints here can help `mypy` to catch potential bugs in other code referencing this property (both in `lightgbm` and in projects relying on `lightgbm`).

Reply:
Ah, completely forgot about that we're trying to add type hints everywhere, sorry.